cms/envs/common.py

```diff
@@ -1115,3 +1120,25 @@
     'options': {},
 }
 PROCTORING_SETTINGS = {}
+
+
+############################ OAUTH2 Provider ###################################
+
+# OpenID Connect issuer ID. Normally the URL of the authentication endpoint.
+OAUTH_OIDC_ISSUER = 'https://www.example.com/oauth2'
```

Comment:
For JWT auth to succeed, this value must match the issuer expected by Programs. At the moment, Programs expects the value of `OAUTH_OIDC_ISSUER` configured for the LMS. Does it make more sense to configure that same value for use in CMS settings (via a PR against configuration-secure), or customize Programs' JWT-DRF `jwt_decode_handler` such that it will accept multiple issuer values?